persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java

```diff
@@ -230,6 +231,10 @@ public static String updateBallerinaToml(Path configPath, String module, String
                         PERSIST_DIRECTORY, null));
                 moduleMembers = populateBallerinaNodeList(moduleMembers, module, datasource);
             }
+            moduleMembers = addNewLine(moduleMembers, 1);
+            moduleMembers = moduleMembers.add(SampleNodeGenerator.createTableArray(
+                    BalSyntaxConstants.PERSIST_DEPENDENCY, null));
+            moduleMembers = populatePersistDependency(moduleMembers);
```

Comment:
Dont we have to check if this is already present or not?

Reply:
Yes, I added now

Reply:
Here you are adding i to the members even if the check is true or not. Wouldnt that lead to duplicates? Is there a testcase for this scenario?

Reply:
Is there a test case for this scenario?

Reply:
Yes. https://github.com/ballerina-platform/persist-tools/pull/152/files#diff-c91429e180e3fbe4d5912d6823485ef3d142a00af6ece2c667f30e25d7c6f157R17